internal/pkg/daemon/selinuxpolicy/selinuxpolicy_controller.go

```diff
@@ -231,13 +254,15 @@ func (r *ReconcileSP) reconcilePolicy(sp *spov1alpha1.SelinuxPolicy, l logr.Logg
 	case installedStatus:
 		policyCopy.Status.State = spov1alpha1.PolicyStateInstalled
 		policyCopy.Status.SetConditions(rcommonv1.Available())
+		r.record.Event(sp, event.Normal(reasonInstalledPolicy, "Successfully wrote policy to disk"))
 	case failedStatus:
 		policyCopy.Status.State = spov1alpha1.PolicyStateError
 		policyCopy.Status.SetConditions(rcommonv1.Unavailable())
+		r.record.Event(sp, event.Warning(reasonCannotInstallPolicy, errors.New("Failed to write policy to disk")))
 	}
 
 	if err := r.client.Status().Update(context.TODO(), policyCopy); err != nil {
-		return reconcile.Result{}, errors.Wrap(err, "Updating SELinux policy with installation success")
+		return reconcile.Result{}, errors.Wrap(err, "Updating SELinux policy with installation result")
```

Comment:
"with installation result" doesn't entirely make sense, is `result` supposed to be a variable? This should only happen on success afaict since `err != nil` above

Reply:
Thanks, you are right this didn't make sense. Luckily the offending code was refactored into the nodeStatus module so the offending line is no longer present in the PR.